torchtune/modules/transformer.py

```diff
@@ -466,16 +466,21 @@ def forward(
         # shape: [b, s, d]
         h = self.norm(h)
 
-        if self.num_output_chunks > 0:
-            # shape: [b, seq_len/num_chunks, out_dim] - out_dim is usually the vocab size
-            # Used with CEWithChunkedOutputLoss. Need to set num_output_chunks in the recipe,
-            # before calling forward. Upcasting it done inside of the loss function.
-            output = [
-                self.output(chunk) for chunk in h.chunk(self.num_output_chunks, dim=1)
-            ]
-        else:
-            # shape: [b, seq_len, out_dim]
-            output = self.output(h).float()
+        # Disable activations offloading locally
+        from torchtune.utils._activations_offloading.offload_ctx_mgr import NoOpManager
```

Comment:
:O 

is there anyway that we can do it outside of the transformer?

for example:
`NoOpWrapper(model.output)`

I think that out transformer class it becoming way too crazy haha

Reply:
or in the context manager ignore a module if module in list_ignore_module